.github/workflows/build.yaml

```diff
@@ -0,0 +1,70 @@
+name: build-pipeline
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Should we have `main` branch instead of `master`?

Reply:
Yes, I didn't checked signoz default branch is main. I have changed it in my latest commit. Thanks